bin/dswx_landcover_mask.py

```diff
@@ -25,14 +26,24 @@ def _get_parser():
         formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 
     # Inputs
-    parser.add_argument('input_file',
+    parser.add_argument('-g', '--ref',
+                        dest='reference_file',
                         type=str,
-                        help='Input HLS product')
-
-    parser.add_argument('-c',
-                        '--copernicus-landcover-100m',
+                        help='Reference file for geographic grid'
+                              ' (e.g. HLS product layer)')
+
+    parser.add_argument('-b', '--bbox',
+                        type=float,
+                        nargs=4,
+                        dest='bbox',
+                        metavar=('LAT_MIN', 'LAT_MAX', 'LON_MIN', 'LON_MAX'),
+                        help='Defines the spatial region in '
+                             'the format south north west east.')
+
```

Comment:
```suggestion
                        metavar=('LON_MIN', 'LAT_MIN', 'LON_MAX', 'LAT_MAX'),
                        help='Defines the spatial bounding box region in '
                             'the NASA STAC-CMR format of LL (lower left) '
                             'and UR (upper right) coordinates, respectively.')

```

Reply:
Thank you, @nemo794 . I decided to remove this script `dswx_landcover_mask.py` from proteus. The script was basically a test script that I made public for John to test our landcover code.